### ISSUE TITLE
Error text: OK (rejected) on home page only

### ISSUE THREAD

**Issue by djohle** [phpmyadmin/phpmyadmin#17636]
I recently upgraded to 5.2.0 and have one small issue that I cannot find any good information about.  On the home page (only) of my installation I get the very generic error (seems to happen for **lot** of different reasons) of:

> Error in processing request
Error code: 200
Error text: OK (rejected)
It seems that the connection to server has been lost. Please check your network connectivity and server status.

I ruled out all the common causes found all over the 'net, and did some further digging.  I narrowed it down to the AJAX error handler, being called from home.js line 133:
`if ($('li.jsversioncheck').length > 0) {
  $.ajax({
    dataType: 'json',
    url: 'index.php?route=/version-check',
    method: 'POST',
    data: {
      'server': CommonParams.get('server')
    },
    success: GitInfo.currentVersion
  });
}`

Seems the actual response from the server is empty, which is maybe why it's considered "rejected" despite having HTTP 200 OK status code.  This is a bit of speculation on my part though.

And to also confirm it is something directly correlated with this "version-check" stuff, it goes away if I add this to my configuration:
> $cfg['VersionCheck'] = false;

All other functionality that I have tried so far (viewing databases, tables, structure, querying data, etc.) seems to work just fine.

**Comment by djohle** [comment]
Oh yeah, and in case it helps, this is running under Apache 2.4.29 & PHP 7.4.30

**Comment by SeVen7nu** [comment]
Error(reject) always emerge when I select a table and click SQL

**Comment by williamdes** [comment]
Hi @djohle 
Sorry for the very late reply to your request :/
Could you try the latest 5.2 version in development [(phpMyAdmin 5.2+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.2+snapshot) ?
And let me know if it works fine please

**Comment by djohle** [comment]
I was working on this today, but just before installing the snapshot I wanted to make sure the problem was still reproducible.  I removed that workaround config setting, and to my surprise it was not failing.

Checking the network requests, I saw that it was in fact doing the "version-check" once again, however that request was now actually coming back with valid JSON (an empty array, no actual information).  As such, the error didn't happen.

This doesn't really "solve" the issue, of course, but I'm going to consider the original problem I had as being transient.

I went ahead with trying the snapshot version as well for good measure, and it is working the same -- no errors.
